manifests/overlays/prod/floorist/cronwf.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: argoproj.io/v1alpha1
+kind: CronWorkflow
+metadata:
+  generateName: solgate-floorist
+  name: solgate-floorist
+spec:
+  schedule: "0 */12 * * *"
+  concurrencyPolicy: "Replace"
+  workflowSpec:
+    arguments:
+      parameters:
+        - name: config
+          value: solgate-floorist
+        - name: split
```

Comment:
What's the motivation behind this being non-default?

Reply:
I guess it was copied from the example PR, shall we make this the default?

Reply:
none, it was an overlook. removed

Reply:
I think it would be better to leave it at default (i.e. remove this parameter name/value entirely) if you don't have a specific reason to limit the number of files each pod pulls. The only scenario where I think changing this value may be desirable is if each file is quite large.

Reply:
done, resolving